src/components/Header.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Form } from './Form';
+import { Todo } from '../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  setTodos: (todos: Todo[] | ((prevTodos: Todo[]) => Todo[])) => void;
+  todosCounter: number;
+  changeComplete: (todo?: Todo) => void;
+};
+
+export const Header: React.FC<Props> = ({
+  todos,
+  setTodos,
+  todosCounter,
+  changeComplete,
+}) => {
+  const handlChangeAllCompleted = () => {
+    changeComplete();
+  };
+
+  return (
+    <header className="todoapp__header">
+      {todos && (
```

Comment:
```suggestion
      {!!todos.length && (
```

Does it work correctly without `length`? Can't check because DEMO link doesn't work((